docs/release/new-release-series.md

```diff
@@ -2,27 +2,31 @@ How to Prepare a New Release Series
 ===================================
 
 !!! warning
```

Comment:
```suggestion
!!!info
```

We don't have to warn people about the source of the notes

Reply:
Right, I guess we don't really need to warn that the instructions were based on Mat's notes ;)